packages/components/src/select-control/test/index.js

```diff
@@ -19,43 +18,43 @@ describe( 'SelectControl', () => {
 	];
 
 	it( 'returns all elements', () => {
-		const selectControl = mount( <SelectControl options={ options } /> );
-		selectControl.setState( {
-			query,
-		} );
+		const { getByRole } = render( <SelectControl options={ options } /> );
 
-		selectControl.instance().search( query );
-		selectControl.update();
+		userEvent.click( getByRole( 'combobox' ) );
 
-		setTimeout( function () {
-			expect(
-				selectControl.find( Button ).filter( '.' + optionClassname )
-					.length
-			).toBe( 3 );
-		}, 0 );
+		expect(
+			getByRole( 'option', { name: 'lorem 1' } )
+		).not.toBeEmptyDOMElement();
+		expect(
+			getByRole( 'option', { name: 'lorem 2' } )
+		).not.toBeEmptyDOMElement();
+		expect(
+			getByRole( 'option', { name: 'bar' } )
+		).not.toBeEmptyDOMElement();
```

Comment:
Not critical, but iterating over options makes sense to me here: `options.forEach( option => expect( getByRole( 'option', { name: option.label } ) ).toBeInTheDocument() );`